### DISCUSSION TITLE
How can I order posts in collection by the published date?

### DISCUSSION THREAD

**Discussion by dasundev** [statamic/cms#7685] [Q&A] [answered]
I am using `published_at` field to order my posts like below:

```antlers
{{ collection:posts orderby (published_at 'desc') }}

    ...

{{ /collection:posts }}
```

But unfortunately, this is not working. 

How can I resolve this?

**Comment by duncanmcclean** [comment, marked as answer]
What you're doing would probably work if you were doing it on an array but `collection:posts` is a tag.

Maybe try this instead:

```antlers
{{ collection:posts sort="published_at:desc" }}
    <!-- ... -->
{{ /collection:posts }}
```

You can review [the Statamic documentation](https://statamic.dev/tags/collection#parameters) to see what parameters are available for the Collection tag.

**Reply by dasundev** [reply]
Thanks so much! It works.
